draft-poidt-ccamp-actn-poi-pluggable-usecases-gaps.md

```diff
@@ -397,40 +402,78 @@ This scenario, shown in {{figure-topo5}}and extends network topologies {{figure-
+Documenting the patch cord that connects the port of the coherent DWDM pluggable in the routers to the optical node (e.g. to the right Add/Drop filter of the ROADM) is performed. This manual operation is prone to human mistakes. It would be highly beneficial for operators to have a mean to check/discover that the right pluggable has been connected to the desired ROADM port. This use case requires the ability to expose to the MDSC the power levels at coherent DWDM pluggable side and at ROADM port side to perform the right correlation and validation.
+
+## Pluggable to pluggable service Provisioning
+The following specific coherent DWDM pluggable provisioning sub-cases are identified:
+* Manual Day 1 configuration (‘valid for coherent pluggable)
+Knowing the coherent pluggable characteristics (performance and optical impairments for a specific operational-mode-ID) optical planning and validation is performed and following parameters are communicated by optical team to IP team: nominal-central-frequency, tx-output-power, operational-mode-ID so that the coherent pluggables at both ends in the routers can be correctly configured (e.g. through P-PNC) in a manual way. As prerequisite before the coherent pluggable configuration, the optical team has properly configured the MC/NMC in the line system DWDM network through the O-PNC.
+*	Semi-manual Day 1 configuration (‘valid for coherent pluggable’)
+Same optical planning and validation is performed first by optical team and then parameters are provided to MDSC operations engineer so that they can be set-up at Hierarchical SDN controller level and provisioned by P-PNC in the corresponding router’s pluggables
+* Semi-Automated Day 1 configuration with Path Computation API request from MDSC towards O-PNC (‘valid for coherent pluggable’) based on trial and error.
+* Fully automated Day 1 configuration (For future discussions)
+
 
-## 3.	End-to-end L3VPN/L2VPN service multi-layer provisioning with SLA constraints (TE constraints) (valid for both)
-This use case is described in {{?I-D.draft-ietf-teas-actn-poi-applicability}} for the SR-TE case which is relevant as target use case for operators. If new connectivity is required between the routers and at optical level then full automation could be achieved. However considering PMO (Present Mode of Operation) in most operators, before an optical path is setup either between two native transponders or between two coherent pluggables in  routers, a detailed optical planning and validation is always required. So, the automation of this use case is considered more for future mode of operations (FMO) and has not the same priority as the previous two use cases
+## 4.	End-to-end L3VPN/L2VPN service multi-layer provisioning with SLA constraints (TE constraints) (valid for both)
```

Comment:
See my comment on line 452. Should be 3 and not 4

Reply:
I removed numbers and let the autonumbering to do the magic